sample/build.gradle

```diff
@@ -30,6 +30,6 @@ dependencies {
     compile 'com.squareup:otto:1.3.8'
     compile 'com.squareup.picasso:picasso:2.5.2'
 
-    compile project(':sdk')
-
+    compile 'com.adzerk:sdk:0.1.0'
+    // compile project(':sdk')
```

Comment:
suggest: reverse the commenting and make `compile project(':sdk')` the *working* dependency.